src-lts/scripts/scr_catspeak_lexer/scr_catspeak_lexer.gml

```diff
@@ -147,8 +147,12 @@ enum CatspeakToken {
     ///
     /// @experimental
     PARAMS = 58,
+    /// The `params_count` keyword.
+    ///
+    /// @experimental
+    PARAMS_COUNT = 59,
     /// Represents a variable name.
-    IDENT = 59,
+    IDENT = 60,
```

Comment:
My idea with the token ids was for them to remain stable, so can you revert all these increments back to their originals, and instead have `PARAMS_COUNT` be 65 and `__SIZE__` be 66.

Reply:
I had a feeling there was a valid reason for it, I'll keep that in mind for the future.